certify

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+unsupportedErr()
+{
+    echo "Invalid Argument! \n"
+    usage
+}
+
+usage()
+{
+    echo "usage: sh certify.sh <ARGS>\n
+    Supported Arguments: 
+    
+    -d | --driverdef -> <String | The path to the Driver Definition YAML file [REQUIRED]>
+    -k | --kubeconfig  -> <String | The path to the kube config file for the cluster being used [Defaults to '/var/run/kubernetes/admin.kubeconfig']
+    -s | --skip      -> <RegExp | A regular expression which will be used to skip matching tests>"
+}
+
+runTests()
+{
+    go test -v ./cmd/... -ginkgo.v -ginkgo.progress --kubeconfig=$kubeconfig --driverdef=$driverDefPath -ginkgo.skip="Pre-provisioned|Inline|Disruptive" -timeout=0
```

Comment:
think you need to add $skipRegExp to end of `-ginko.skip=` line

Reply:
done